### PR TITLE
Add Stripped Logs and Stripped Woods Tags

### PR DESCRIPTION
**Pull request by IThundxr**
Tags for stripped logs and woods, my usecase here is to replace the tags create maintains itself:

[Create's Modded Stripped Logs Tag](<https://github.com/Fabricators-of-Create/Create/blob/mc1.20.1/dev/src/generated/resources/data/create/tags/items/modded_stripped_logs.json>)
[Create's Modded Stripped Woods Tag](<https://github.com/Fabricators-of-Create/Create/blob/mc1.20.1/dev/src/generated/resources/data/create/tags/items/modded_stripped_wood.json>)

NeoForge PR: https://github.com/neoforged/NeoForge/pull/1584

### CODE REVIEW

**Comment by gniftygnome** [comment]
Double-pluralizing the wood tag is a newer thing (which I'll update to match) but aside from that, all three Terraformers biome mods already use these tags, and I see our logs are not already in the Create tags. So, obviously, I'm in support of this PR.

**Review by TelepathicGrunt — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

File: fabric-convention-tags-v2/src/generated/resources/assets/fabric-convention-tags-v2/lang/en_us.json [hunk 1]
```diff
@@ -76,6 +76,8 @@
   "tag.block.c.storage_blocks.redstone": "Redstone Storage Blocks",
   "tag.block.c.storage_blocks.slime": "Slime Storage Blocks",
   "tag.block.c.storage_blocks.wheat": "Wheat Storage Blocks",
+  "tag.block.c.stripped_logs": "Stripped Log Blocks",
```
Comment (Shnupbups) [inline fabric-convention-tags-v2/src/generated/resources/assets/fabric-convention-tags-v2/lang/en_us.json]: Can probably just be `Stripped Logs`

File: fabric-convention-tags-v2/src/generated/resources/assets/fabric-convention-tags-v2/lang/en_us.json [hunk 2]
```diff
@@ -275,6 +277,8 @@
   "tag.item.c.storage_blocks.slime": "Slime Storage Blocks",
   "tag.item.c.storage_blocks.wheat": "Wheat Storage Blocks",
   "tag.item.c.strings": "Strings",
+  "tag.item.c.stripped_logs": "Stripped Log Blocks",
```
Comment (Shnupbups) [inline fabric-convention-tags-v2/src/generated/resources/assets/fabric-convention-tags-v2/lang/en_us.json:280]: same

**Comment by apple502j** [comment]
How does Convention Tags handle experiments, btw?

**Comment by TelepathicGrunt** [comment]
@apple502j wdym? The new wood? Add those as optional entries so if not present, the tag still works.

**Comment by apple502j** [comment]
@TelepathicGrunt The item is always present, it's just that you shouldn't be using them if the experiment is turned off. In vanilla that's done by having another data pack.

**Pull request merged**